### DISCUSSION TITLE
RTC convience macros in the ulp repository?

### DISCUSSION THREAD

**Discussion by kjm1102** [micropython/micropython#12857] [Libraries & Drivers] [answered]
https://github.com/micropython/micropython-esp32-ulp mentions 'RTC convenience macros (e.g. WRITE_RTC_REG)' in the features section but I can't seem to find any usage examples in the docs? Anybody know how these work?

**Comment by cwalther** [comment, marked as answer]
They are documented here: https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/ulp_instruction_set.html#convenience-macros-for-peripheral-registers-access
